flocker/node/agents/gce.py

```diff
@@ -194,10 +201,15 @@ def get_metadata_path(path):
     :returns unicode: The resulting value from the metadata server.
     """
     timeout_sec = 3
-    r = requests.get(_METADATA_SERVER + path,
-                     headers=_METADATA_HEADERS,
-                     timeout=timeout_sec)
-    return r.text
+    with start_action(
+            action_type=u"flocker:node:agents:gce:get_metadata_path",
+            path=path
+    ) as action:
+        r = requests.get(_METADATA_SERVER + path,
+                         headers=_METADATA_HEADERS,
+                         timeout=timeout_sec).text
+        action.add_success_fields(response=r)
```

Comment:
Done.

I sort of don't like having r.text on 2 lines in a row, but I think I'll just get over that.